src/us/kbase/workspace/test/workspace/WorkspaceListenerTest.java

```diff
@@ -26,6 +26,7 @@
 import com.google.common.collect.ImmutableMap;
```

Comment:
Lots of cleanup possible for this file, but the current cleanup helps a lot and is 400 lines so... later